src/ripple/app/tests/HashRouter_test.cpp

```diff
@@ -171,14 +181,10 @@ class HashRouter_test : public beast::unit_test::suite
     void
     run()
     {
-        UptimeTimer::getInstance().beginManualUpdates();
-
         testExpiration();
         testSuppression();
         testSetFlags();
         testSwapSet();
-
-        UptimeTimer::getInstance().endManualUpdates();
     }
 };
 
```

Comment:
I updated the `testExpiration` to account for the new behavior, but that's a good point that it doesn't really show stuff _not_ expiring when it gets hit. So I added a new `testNonExpiration`, which is a copy of the original test, with the `expect`s updated for the new behavior.